cmd/src/code_intel_upload_flags.go

```diff
@@ -320,28 +208,12 @@ func inferDefaultFile() (string, error) {
 		}
 	}
 
-	hasLSIF := true
-	const lsifFilename = "dump.lsif"
-	if _, err := os.Stat(lsifFilename); err != nil {
-		if os.IsNotExist(err) {
-			hasLSIF = false
-		} else {
-			return "", err
-		}
-	}
-
-	if hasSCIP && hasLSIF {
-		return "", errors.Newf("both %s and %s exists - cannot determine unambiguous choice", scipFilename, lsifFilename)
-	}
-	if !(hasSCIP || hasLSIF) {
-		return "", formatInferenceError(argumentInferenceError{"file", errors.Newf("neither %s nor %s exists", scipFilename, lsifFilename)})
-	}
-
-	if hasSCIP {
+	if !hasSCIP {
+		return "", formatInferenceError(argumentInferenceError{"file", errors.Newf("%s does not exist", scipFilename)})
+	} else {
```

Comment:
I think you could simplify this function a bit to avoid the `hasSCIP` boolean.

```
const scipFilename = "index.scip"
_, err := os.Stat(scipFilename)
if err == nil {
    return scipFilename, nil
}
if os.IsNotExist(err) {
    return "", <construct special error here>
}
return "", err
```